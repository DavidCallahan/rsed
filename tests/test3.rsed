print "variable test 1"
set $me = "foo"
print $me
print "line=" $LINE
copy 1
print "line=" $LINE

