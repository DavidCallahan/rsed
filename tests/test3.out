variable test 1
foo
line=0
1
line=2
